Updates to CREATE DATABASE for Cypher 25 (#2132)
neo4j/docs-operations: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

[[manage-databases-create-database-options]]
=== Options

[.tabbed-example]
=====

> [.include-with-cypher5]

Reviewer: ```suggestion
[.include-with-cypher-5]
```

If this change is applied to this and other `include-with...` lines, it will update the generated tab title from eg `Cypher5` and `Cypher25` to `Cypher 5` etc (space between `Cypher` and `5`).

For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#set-enrichment-mode/[Change Data Capture Manual -> Enable CDC on self-managed instances -> Set the enrichment mode].

| `storeFormat`
| `aligned` \| `standard` \| `high_limit` \| `block`
|
Defines the store format if the database created is new.
`high_limit` and `standard` formats are deprecated from 5.23.
For more information on store formats, see xref::database-internals/store-formats.adoc[Store formats].

If the store is seeded with `seedURI` or `existingDataSeedServer`, or if the command is used to mount pre-existing store files already present on the disk, they will retain their current store format without any modifications.

| `seedRestoreUntil`
| Datetime or transaction id. E.g. `datetime("2025-01-01T12:15:00.000+0100")` or `123456`
| 
If you are passing a `seedURI` that leads to a backup chain, including differential backups, you can choose to not apply all the transactions in the differential backups.
To seed up to a specific date, specify a `datetime` and this will seed the database with transactions committed before the provided timestamp.

> To seed up to a specific transaction ID, specify a transaction id, and this will seed the database with transactions up to, but not including the specified transaction.

Reviewer 2: ```suggestion
To seed up to a specific transaction ID, specify a transaction ID.
This will seed the database with transactions up to, but not including the specified transaction.
```

The `success` column provides an aggregate status of whether or not the command is considered successful.
Thus, every row has the same value, determined on a successful completion without a timeout.
======
[.include-with-cypher-25]
======

> Sub-clause `WAIT` allows you to specify a time limit in which the command must complete and return.

Reviewer 2: ```suggestion
Sub-clause `WAIT` allows you to specify a time limit for the command to complete and return.
```

Author: This is the same in the Cypher 5 section above also.

[source, cypher]
----
CREATE DATABASE slow WAIT 5 SECONDS
----

.Result
[role="queryresult"]
----
info: Server `ServerId{b55c6551}` at address `server1:7687` has caught up.
03N85 (Neo.ClientNotification.Cluster.ServerCaughtUp)

info: Server `ServerId{a9e7e8f1}` at address `server2:7687` has caught up.
03N85 (Neo.ClientNotification.Cluster.ServerCaughtUp)

info: Server `ServerId{0f7cb48e}` at address `server3:7687` has caught up.
03N85 (Neo.ClientNotification.Cluster.ServerCaughtUp)
----

> The command will return a notification for each server in the cluster to indicate the status of that command

Reviewer 2: ```suggestion
The command returns a notification for each server in the cluster to indicate the status of that command on that server.
```

> on that server. In this case all three cluster members have returned `Neo.ClientNotification.Cluster.ServerCaughtUp`

Reviewer 2: ```suggestion
In this example, all three cluster members have returned `Neo.ClientNotification.Cluster.ServerCaughtUp`, which indicates that the server has applied the command successfully and is up to date.
```

> which indicates that the server has applied the command successfully and is up to date.

Reviewer 2: ```suggestion
```
